Makefile

```diff
@@ -148,7 +153,7 @@ clear-protos:
 
 ### Check vendored files
 .PHONY: vendor-check
-vendor-check: clear-protos install-tools vendor-dependencies
+vendor-check: clear-protos vendor-dependencies
```

Comment:
It looks like install tools only pulls proto related tools.  Remove it?

Reply:
Good point, we no longer need that.